### ISSUE TITLE
CUSTMOZIABLE PLOTS not appeared

### ISSUE THREAD

**Issue by thw1023** [abdeladim-s/tensorboard_plugin_customizable_plots#6]
The new `TAB CUSTMOZIABLE PLOTS` did not appear and no error was reported for installation. What could be the cause?

**Comment by absadiki** [comment]
@thw1023, have you followed the installation correctly ?

**Comment by thw1023** [comment]
Yes, the installation went smoothly without any errors, and the plug-in already exists in the environment.













At 2023-04-21 06:45:32, "Abdeladim Sadiki" ***@***.***> wrote:

@thw1023, have you followed the installation correctly ?

**Comment by thw1023** [comment]
Thanks for your reply. Due to time, I have chosen to download and redraw the data to add the legend.
















At 2023-04-21 06:45:32, "Abdeladim Sadiki" ***@***.***> wrote:

@thw1023, have you followed the installation correctly ?

**Comment by marawangamal** [comment]
I followed the installation correctly as well. For me the CUSTOMIZABLE PLOTS tab appears, but is an empty page

**Comment by Soha18** [comment]
Hi, I had the same issue, how did you solve it?  please advice
Thanks

**Comment by thw1023** [comment]
Due to time, I have chosen to download and redraw the data to add the legend.



---- Replied Message ----
| From | ***@***.***> |
| Date | 07/14/2023 22:37 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [abdeladim-s/tensorboard_plugin_customizable_plots] CUSTMOZIABLE PLOTS not appeared (Issue #6) |

Hi, I had the same issue, how did you solve it? please advice
Thanks

**Comment by Luke-Mason** [comment]
Please fix the issue, The tab does not appear and so this plugin is still broken

**Comment by Luke-Mason** [comment]
> Please fix the issue, The tab does not appear and so this plugin is still broken

Nevermind i fixed the issue, tensorboard was installed in a different environment, where as this plugin was installed in another. (they existed in different site-packages/ paths) I recommend uninstalling tensorboard, then reinstalling it with this plugin again to ensure that pip is installing in your environment and not seeing that the requirement is met in a different maybe local environment

**Comment by absadiki** [comment]
> > Please fix the issue, The tab does not appear and so this plugin is still broken
> 
> Nevermind i fixed the issue, tensorboard was installed in a different environment, where as this plugin was installed in another. (they existed in different site-packages/ paths) I recommend uninstalling tensorboard, then reinstalling it with this plugin again to ensure that pip is installing in your environment and not seeing that the requirement is met in a different maybe local environment

Yes both packages need to be in the same virtual environment so that Tensorboard can find the plugin files. 
Thanks @Luke-Mason for posting your solution, It will certainly help others facing the same issue.
